tasks/sep/032-op-pectra-defence/VALIDATION.md

```diff
@@ -0,0 +1,126 @@
+  changes occur.
+- All addresses (in section headers and storage values) match the provided name, using the Etherscan and Superchain
+  Registry links provided. This validates the bytecode deployed at the addresses contains the correct logic.
+- All key values match the semantic meaning provided, which can be validated using the storage layout links provided.
+
+## Nested Safe State Overrides and Changes
+
+This task is executed by the nested 2/2 `ProxyAdminOwner` Safe. Refer to the
+[generic nested Safe execution validation document](../../../NESTED-VALIDATION.md)
+for the expected state overrides and changes.
+
+The `approvedHashes` mapping **key** of the `ProxyAdminOwner` that should change during the simulation is
+- Council simulation: `0x3394b41f3c05f4c1bb2767a59f18fd9cc4497d7a8781dca9206396044c5d4a80`
+- Foundation simulation: `0xfd5f9ae7ec744b3e9bbfd555849eb2f5727c46fe3e5cd532c7e102c0934e99c1`
+
+calculated as explained in the nested validation doc.
+
+Additionally, the nonces [will increment by one](../../../NESTED-VALIDATION.md#nonce-increments).
+
+## State Changes
+
+Note: The changes listed below do not include safe nonce updates or liveness guard related changes. Refer to the
+[generic nested Safe execution validation document](../../../NESTED-VALIDATION.md)
+
+### `0x05F9613aDB30026FFd634f38e5C4dFd30a197Fa1` (`DisputeGameFactoryProxy`)
+
+Click on 'show raw state changes'.
+
+- **Key**: `0x4d5a9bd2e41301728d41c8e705190becb4e74abe869f75bdb405b63716a35f9e` <br/>
+  **Before**: `0x000000000000000000000000bbdbdfe37c02439764de0e41c906e4396b5b3914` <br/>
+  **After**: `0x0000000000000000000000006b8a1c71e23e103d3a81e7878656be9516733ca3` <br/>
+  **Meaning**: Updates the implementation for game type 1. Verify that the old implementation is set in this slot using
+  `cast call 0x05F9613aDB30026FFd634f38e5C4dFd30a197Fa1 "gameImpls(uint32)(address)" 1`.
+
+- **Key**: `0xffdfc1249c027f9191656349feb0761381bb32c9f557e01f419fd08754bf5a1b` <br/>
+  **Before**: `0x000000000000000000000000f3ccf0c4b51d42cfe6073f0278c19a8d1900e856` <br/>
+  **After**: `0x000000000000000000000000927248cb1255e0f02413a758899db4aecffaa5fe` <br/>
+  **Meaning**: Updates the implementation for game type 0. Verify that the old implementation is set using
+  `cast call 0x05F9613aDB30026FFd634f38e5C4dFd30a197Fa1 "gameImpls(uint32)(address)" 0`.
+
+
+## Verifying Dispute Games
+
+<!-- Sepolia todo: We should convert this to solidity and run the checks in the postCheck method -->
+The old and new dispute game contracts can be compared with the [comparegames.sh](https://gist.github.com/ajsutton/28be852a36d9d19af16f7c870b267873)
+script.
+
+From the tenderly simulation click on 'Run on Fork', then copy and export the RPC url provided.
+
+```
+export ETH_RPC_URL=https://rpc.tenderly.co/fork/...
+```
+
+The arguments to the script can be taken from the before and after values in the `DisputeGameFactoryProxy` above.
+
+### PermissionedDisputeGame:
```

Comment:
We should just give the two commands, a bit tedious to extract the addresses from the 32 byte slot